kratos/includes/kratos_components.h

```diff
@@ -137,22 +131,23 @@ class KRATOS_API(KRATOS_CORE) KratosComponents
     ///@{
 
     /// Turn back information as a string.
-    virtual std::string Info() const
+    static std::string Info()
```

Comment:
Why are you making those static? Not that I'm complaining, but note that this breaks the `__str__` printing in python if you had it.

Reply:
Why it breaks the __str__ printing? We use the print object, isn't it?

Reply:
the `__str__` mechanism is for printing class instances, so it expects a self. Once the method is static, it won't get the self. I guess that the python method can be made non-static, but then you need an instance of KratosComponents to print.